lib/clients/CatalogClient.py

```diff
@@ -0,0 +1,1074 @@
+# -*- coding: utf-8 -*-
```

Comment:
Did we move this file?  (This would explain my earlier comment.)

Reply:
I suspect that this code didn't used to be in the repo, and it was copied out of the old gigantic dev container.